### ISSUE TITLE
Logging both weighted and unweighted VAL loss

### ISSUE THREAD

**Issue by Jubeku** [ecmwf/WeatherGenerator#743]
### Is your feature request related to a problem? Please describe.

Both weighted and unweighted validation loss should be logged for ease of comparisons. Currently only one of them is reported, and there is not clear from the logs if the loss is weighted or not. 

### Describe the solution you'd like

Log both weighted and unweighted validation losses and name them explicitly. 

### Organisation

ECMWF

**Comment by clessig** [comment]
We could potentially also have score-like validation functions, e.g. rmse_nhem.
